src/app/store/manage-tokens/manage-tokens.slice.ts

```diff
@@ -0,0 +1,42 @@
+import { useSelector } from 'react-redux';
+
+import { PayloadAction, createEntityAdapter, createSelector, createSlice } from '@reduxjs/toolkit';
+
+import type { RootState } from '..';
+
+interface TokenUserSetting {
+  id: string;
+  enabled: boolean;
+  accountIndex: number;
+}
+
+const manageTokensAdapter = createEntityAdapter<TokenUserSetting>();
+
+export const manageTokensSlice = createSlice({
+  name: 'manageTokens',
+  initialState: manageTokensAdapter.getInitialState(),
+  reducers: {
+    setToken(state, action: PayloadAction<{ id: string; enabled: boolean; accountIndex: number }>) {
```

Comment:
Can we model over the action/event taking place? https://redux.js.org/style-guide/#model-actions-as-events-not-setters

Reply:
Agree here, trying to understand some of the changes, is it nec to have `manage` in the naming? Isn't a slice of state always `managing` state, would just `tokensSlice` be sufficient here to describe it? I think there might be a way to make what is happening here more clear, like what is `setToken` doing?